module/applications/fields.mjs

```diff
@@ -0,0 +1,12 @@
+/**
+ * Create a text input for a StringField.
+ * @param {StringField} field               The field.
+ * @param {FormInputConfig<string>} config  The input configuration.
+ * @returns {HTMLElement|HTMLCollection}
+ */
+export function createTextInput(field, config) {
```

Comment:
Basically only using this to polyfill the core `createTextInput` and allow attaching a class to the input.